specs/trust-store-trust-policy.md

```diff
@@ -414,10 +411,11 @@ When delta CRLs are implemented, the following results can occur during revocati
 
 ##### OCSP Download
 
-OCSP URLs can be obtained from the certificate's authority information access (AIA) extension as defined in [RFC-2560](https://datatracker.ietf.org/doc/html/rfc2560).
-Notary V2 will wait for a default threshold of 5 seconds to receive an OCSP response.
-If OCSP response is not available within the timeout threshold the revocation result will be "revocation unavailable".
+OCSP URLs can be obtained from the certificate's authority information access (AIA) extension as defined in [RFC 6960](https://www.rfc-editor.org/rfc/rfc6960).
+If the certificate contains multiple OCSP URLs, then each URL is invoked in sequential order.
+For each OCSP URL, Notary V2 will wait for a default threshold of 5 seconds to receive an OCSP response.
```

Comment:
Can you provide some details on how we arrived at 5 seconds? Why can't it be lower?

s/Notary v2/Notation

Reply:
We can have it lower; just for consistency between crl and ocsp I picked 5 sec. Do you have any recommendation?

Reply:
As per discussion with Milind will update ocsp timeout to 2 seconds